deploy/cr.yaml

```diff
@@ -145,7 +145,39 @@ spec:
   port: "5432"
 # The number of replicas to create after a PostgreSQL primary
 # is first initialized. This only works on create
-  replicas: "0"
+#  replicas: "0"
+  pgReplicas:
+    hotStandby:
+      size: 0
+#      resources:
+#        requests:
+#          cpu: "1"
+#          memory: "128Mi"
+#        limits:
+#          cpu: "1"
+#          memory: "128Mi"
+#      volumeSpec: 
+#        accessmode: ReadWriteOnce
+#        matchLabels: ""
+#        name: ""
+#        size: 1G
+#        storageclass: ""
+#        storagetype: dynamic
+#        supplementalgroups: ""
+#      affinity:
+#        default: null
+#      labels:
+#        pg-cluster-label: cluster1
+#      annotations:
+#        pg-cluster-annot: cluster1
+#      enableSyncStandby: false
+#      expose:
+#        serviceType: LoadBalancer
+#        loadBalancerSourceRanges:
+#        annotations:
```

Comment:
annotations do not work

Reply:
https://github.com/percona/percona-postgresql-operator/pull/50/commits/35d324887677775fcc5696fb0d5c41290cdd92d0